friends/src/components/Friends/FriendList.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import FriendDetails from './FriendDetails';
+
+const  FriendList = ({ friends, updateFriend, deleteFriend, ...props }) => {
+  return (
+    <div>
+    {
+      friends.map(friend => (
+        <FriendDetails
+              {...props}
+              key={friend.id}
+              updateFriend={updateFriend}
+              deleteFriend={deleteFriend}
+              friends={friend}
+       />
+       ))
+    }
+    </div>
+  )
+}
+
+
+// export class FriendList extends React.Component {
```

Comment:
After refactoring the code you can remove the commented parts and only left some comments of what you changed if that helps you